utils/build-script

```diff
@@ -662,6 +711,12 @@ the number of parallel build jobs to use""",
             "--skip-test-watchos",
         ]
 
+    try:
+        compiler = CompilerExecutable(args)
```

Comment:
Let's call the variable `host_compiler`.  I know other contributors are working on cross-compilation support.